source/MQTTFileDownloader.c

```diff
@@ -66,10 +76,16 @@ static uint16_t createTopic( char * topicBuffer,
  * @param[in] messageLength Length of the MQTT message received.
  * @param[out] decodedData Buffer to place the decoded data in.
  * @param[out] decodedDataLength Length of decoded data.
+ * @param[out] fileId ID of the file to which the data block belongs.
+ * @param[out] blockId ID of the received block.
+ * @param[out] blockSize Size of the receive block in bytes.
  *
  * @return uint8_t returns appropriate MQTT File Downloader Status.
  */
-static MQTTFileDownloaderStatus_t handleCborMessage( uint8_t * decodedData,
+static MQTTFileDownloaderStatus_t handleCborMessage( int32_t * fileId,
+                                                     int32_t * blockId,
+                                                     int32_t * blockSize,
+                                                     uint8_t * decodedData,
```

Comment:
Super-nit: In general, I'm a fan of putting input params before output params

Reply:
Fixed with https://github.com/aws/aws-iot-core-mqtt-file-streams-embedded-c/pull/23/commits/0a7043b4b1f23ed011934f485f0afa72d0c7fb5c.